pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md

```diff
@@ -51,5 +57,59 @@ Calico is now pre-installed in a DC/OS cluster and can be used by containers to
 Metronome based jobs can now join container networks to communicate with other services/jobs in the same network.
 
 ## Domain Sockets for Agent Executor Communication
-
 Agents and Executors now communicate over Unix Domain sockets making operators life easy in the presence of container overlay networks.
+### Deprecation of Marathon LB
+
+### Hive Metastore for Spark SQL
```

Comment:
I would actually change this to Deprecation of Hive Metastore. cc @kaiwalyajoshi. 
Also, this is a package release. So I wouldn't call it here?

Reply:
What do you mean a package release?

Reply:
Ok took this out of release notes.